transforms/universal/web2parquet/README.md

```diff
@@ -0,0 +1,33 @@
+# Web Crawler to Parquet
+
+This tranform crawls the web and downloads files in real-time.
+
+This first release of the transform, only accepts the following 4 parameters. Additional releases will extend the functionality to allow the user to specify additional constraints such as mime-type, domain-focus, etc.
+
+
+## Parameters
+
+For configuring the crawl, users need to identify the follow parameters:
+
```

Comment:
identify the follow -> specify the following

Reply:
You are having fun! aren't you? lol